### PR TITLE
products and categories written , Category bar scheme written

### PR DESCRIPTION
**Pull request by ibrahimsugun**
Not finish but gone so many miles

### CODE REVIEW

**Review by ibrahimsugun — COMMENTED** [review]
good work

**Review by ibrahimsugun — COMMENTED** [review]
waiting for rewiev confirm

**Comment by KerimCETINBAS** [comment]
Cannot resolve confilitcts

File: apps/site/src/routes/Footer.svelte [hunk 1]
```diff
@@ -29,7 +29,7 @@
                         <a class="text-dark mb-2" href="index.html"><i class="fa fa-angle-right mr-2"></i>Home</a>
                         <a class="text-dark mb-2" href="shop.html"><i class="fa fa-angle-right mr-2"></i>Our Shop</a>
                         <a class="text-dark mb-2" href="detail.html"><i class="fa fa-angle-right mr-2"></i>Shop Detail</a>
-                        <a class="text-dark mb-2" href="cart.html"><i class="fa fa-angle-right mr-2"></i>Shopping Cart</a>
+                        <a class="text-dark mb-2" href="cart.html"><i class="fa fa-angle-right mr-2"></i>sShopping Cart</a>
```
Comment (KerimCETINBAS) [inline apps/site/src/routes/Footer.svelte:32]: has typo sShopping card

**Review by KerimCETINBAS — APPROVED** [review]

**Pull request merged**